src/Microsoft.DotNet.TemplateLocator/Microsoft.DotNet.TemplateLocator.csproj

```diff
@@ -12,8 +12,17 @@
     <EmitLegacyAssetsFileItems>true</EmitLegacyAssetsFileItems>
 
     <PackageId>microsoft.dotnet.templateLocator</PackageId>
+
+    <!--https://github.com/NuGet/Home/issues/3891#issuecomment-377319939-->
+    <TargetsForTfmSpecificBuildOutput>$(TargetsForTfmSpecificBuildOutput);CopyProjectReferencesToPackage</TargetsForTfmSpecificBuildOutput>
   </PropertyGroup>
 
+  <Target Name="CopyProjectReferencesToPackage" DependsOnTargets="ResolveReferences">
+    <ItemGroup>
+      <BuildOutputInPackage Include="@(ReferenceCopyLocalPaths->WithMetadataValue('ReferenceSourceTarget', 'ProjectReference'))" />
```

Comment:
Who/what is consuming this item group? The targetname suggests it's going to copy something, but it seems to just create/add items

Reply:
This is the workaround provided by nuget team https://github.com/NuGet/Home/issues/3891#issuecomment-377319939 to include package reference to dll. Nuget targets are consuming this gruop